vega_sim/scenario/hedged_market_maker/scenario.py

```diff
@@ -0,0 +1,333 @@
+            initial_asset_mint=1e10,
+            commitment_amount=1e6,
+            asset_name=self.asset_name,
+            asset_decimal_places=self.asset_adp,
+            market_name=self.ext_name,
+            market_decimal_places=self.ext_mdp,
+            num_steps=self.num_steps,
+            num_levels=25,
+            kappa=self.ext_mm_kappa,
+            market_kappa=self.ext_mm_market_kappa,
+            tick_spacing=self.ext_mm_tick_spacing,
+            inventory_upper_boundary=30,
+            inventory_lower_boundary=-30,
+            state_update_freq=60,
+            fee_amount=1e-03,
+        )
+
+        int_auction_pass_bid = OpenAuctionPass(
+            wallet_name=AUCTION_PASS_BID.wallet_name,
+            wallet_pass=AUCTION_PASS_BID.wallet_pass,
+            key_name=AUCTION_PASS_BID.key_name,
+            side="SIDE_BUY",
+            initial_asset_mint=1e10,
+            initial_price=initial_price,
+            market_name=self.int_name,
+            asset_name=self.asset_name,
+            opening_auction_trade_amount=self.int_pdp,
+        )
+        int_auction_pass_ask = OpenAuctionPass(
+            wallet_name=AUCTION_PASS_ASK.wallet_name,
+            wallet_pass=AUCTION_PASS_ASK.wallet_pass,
+            key_name=AUCTION_PASS_ASK.key_name,
+            side="SIDE_SELL",
+            initial_asset_mint=1e10,
+            initial_price=initial_price,
+            market_name=self.int_name,
+            asset_name=self.asset_name,
+            opening_auction_trade_amount=self.int_pdp,
+        )
+        ext_auction_pass_bid = OpenAuctionPass(
+            wallet_name=AUCTION_PASS_BID.wallet_name,
+            wallet_pass=AUCTION_PASS_BID.wallet_pass,
+            key_name=AUCTION_PASS_BID.key_name,
+            side="SIDE_BUY",
+            initial_asset_mint=1e10,
+            initial_price=initial_price,
+            market_name=self.ext_name,
+            asset_name=self.asset_name,
+            opening_auction_trade_amount=self.ext_pdp,
+        )
+        ext_auction_pass_ask = OpenAuctionPass(
+            wallet_name=AUCTION_PASS_ASK.wallet_name,
+            wallet_pass=AUCTION_PASS_ASK.wallet_pass,
+            key_name=AUCTION_PASS_ASK.key_name,
+            side="SIDE_SELL",
+            initial_asset_mint=1e10,
+            initial_price=initial_price,
+            market_name=self.ext_name,
+            asset_name=self.asset_name,
+            opening_auction_trade_amount=self.ext_pdp,
+        )
+
+        int_random_trader = MarketOrderTrader(
+            wallet_name=INT_RANDOM_TRADER.wallet_name,
+            wallet_pass=INT_RANDOM_TRADER.wallet_pass,
+            key_name=INT_RANDOM_TRADER.key_name,
+            market_name=self.int_name,
+            asset_name=self.asset_name,
+            initial_asset_mint=1e10,
+            buy_intensity=1 * 10 ** (self.int_pdp),
+            sell_intensity=1 * 10 ** (self.int_pdp),
+            base_order_size=1 * 10 ** -(self.int_pdp),
```

Comment:
Might need to make these configurable again at some point

Reply:
I added these agents to provide a small trade at each step to guarantee candle data. Think I left these out to try avoid having too many arguments in the scenario. Happy to add arguments for these in the future though if we see another use for the `MarketOrderTraders`.

Reply:
That makes sense, forgot about that purpose for them! Probably no need to have them at all configurable no